pom.xml

```diff
@@ -292,6 +287,13 @@
         </dependencies>
     </dependencyManagement>
 
+    <dependencies>
```

Comment:
The dependency is already included in Line 242, but with a specific version.
Is there a reason why it's mentioned twice?

Reply:
One is for dependency management (to have the same version in each of the sub modules), the other one is to provide the dependency.